plugins/mysql/mysql_disk_by_prefix

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+# -*- python -*-
+
+"""
+=head1 INTRODUCTION
+
+Plugin to monitor MySQL database disk usage per prefix
+
```

Comment:
Please add a section `APPLICABLE SYSTEMS` (which is very common for munin plugins) and mention that local access to the database directory is required.

Reply:
Done.